rockcraft/models/project.py

```diff
@@ -87,9 +87,24 @@
     ),
 ]
 
-BaseT = Literal["bare", "ubuntu@20.04", "ubuntu@22.04", "ubuntu@24.04"]
+BaseT = Literal[
+    "bare",
+    "ubuntu@20.04",
+    "ubuntu@22.04",
+    "ubuntu@24.04",
+    "ubuntu@25.04",
```

Comment:
What's the reasoning for adding plucky? Can we skip it?

Reply:
The maven spread test uses chisel, which does not yet have a questing branch.

Per separate conversation I'll remove plucky, turn off that test for plucky and add a task to turn it on for questing.

Reply:
https://github.com/canonical/rockcraft/issues/953